docs/java-rest/high-level/ml/update-job.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[java-rest-high-x-pack-ml-update-job]]
+=== Update Job API
+
+The Update Job API provides the ability to update a {ml} job.
+It accepts a `UpdateJobRequest` object and responds
+with a `PutJobResponse` object.
+
+[[java-rest-high-x-pack-ml-update-job-request]]
+==== Update Job Request
+
+A `UpdateJobRequest` object gets created with an `JobUpdate` object.
```

Comment:
`an` -> `a`

Reply:
I should just always make `The` so there is a certain authority and exactness to it :)